src/hooks/resourceHooks/useLocalGokbPkg.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { PACKAGES_ENDPOINT } from '../../constants';
+
+const useLocalGokbPkg = ({ reference }) => {
+  const ky = useOkapiKy();
+  const remoteId = reference ? reference.split(':')[0] : null;
+  const { data } = useQuery(
+    ['PKG', 'fetchLocalPkg', remoteId],
+    () => ky.get(`${PACKAGES_ENDPOINT}?filters=identifiers.identifier.value==${remoteId}&&identifiers.identifier.ns.value==gokb_id&&identifiers.identifier.status.value==approved`).json(),
+    {
+      enabled: !!remoteId,
+    }
+  );
+
+  return data?.[0];
```

Comment:
We normally have options to return either the full query, or the data at hand. I'd prefer us try to grab the first one as one named export, but have the full data available for debugging/futureProofing (again though... I think this is a wrapping of usePackage)